jobs/ruby/senior-ruby-developer.md

```diff
@@ -1,22 +1,19 @@
-# Middle/Senior Ruby Developer
+# Senior Ruby Developer
 
-We're looking for a pro-active middle/senior-level web developer with experience in Ruby and Ruby on Rails, who not only solves tasks but suggests improvements on the codebase level as well as on product and company level, and who shares knowledge within the team on meetups and in chats. Check out the [video](https://www.youtube.com/watch?v=kc1jcy00AJ0&feature=youtu.be) with Andrew (who is currently responsible for this role), where he tells about the project and role itself. <b> We are open to a person with ambitions to fill in the Team Lead role. </b>
+We're looking for a pro-active senior-level web developer with experience in Ruby and Ruby on Rails, who not only solves tasks but suggests improvements on the codebase level as well as on product and company level, and who shares knowledge within the team on meetups and in chats. <b> We are open to a person with ambitions to fill in the Team Lead role. </b>
 
 ## About the job
 
-- You'll work on a project with a team of junior/middle developers (2-4 people). Below is the list of technologies we use for this project. You'll have the opportunity to work with them while working with us. It's fine if you don't know some technologies from the list, but we expect you to learn them after you start to work. Here is the list:
-  - Ruby on Rails / GraphQL / PostgreSQL / ElasticSearch
-  - RSpec / Minitest / Capybara
-  - JavaScript (ES6 and beyond) / React / React Relay / CSS Modules
-  - Flow Types / Webpack / Jest
-  - Docker / Kubernetes / AWS
-- Salary 𝚫: 30k—54k USD.
-- For development we usually use Kanban or similar process in Trello, Pivotal of Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use Holacracy for organizational governance.
+- Salary 𝚫: 2500-4500USD.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually, our projects last for about 1.5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- For development we usually use Kanban or similar process in Trello, Pivotal or Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use Holacracy for organizational governance.
 - Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
 - You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
 
 ## We expect that you
 
+- Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important
```

Comment:
The "it's very important" part looks odd. Sounds more like a constraint rather that a foundation for great work. And as if we have a lot of issues with this part. It either case I wouldn't advertise it this way. As I read it it's already under "We expect that you ..." so such extra bits seem somewhat redundant.

Reply:
@hesalx what do you suggest to make it not odd while I'm indeed looking for a constraint that will filter out candidates having unmatched with our core values?

I really see no reason to spend anyone's time on a few rounds of interviews, then decision-making meetings, then a trial period, and then having difficult talks with clients, etc etc. We've indeed had a couple of noticeable cases of hiring in 2021 when we had to put our ways quickly because of cultural unmatch.

Reply:
@atatarenko @hesalx  "We expect" we probably can change to "What we’d like to see in your background & skills we think are necessary" if that makes any sense, maybe that sounds a bit softer, but in any case this is the eye catching area for all the candidates and everyone understands that here will go the requirements

Reply:
@dashandrina if this is not a string preference, I'd pitch for keeping it as-is for the sake of a more informative style.

Reply:
@atatarenko I totally share your concern but I also care for the brand :) Current phasing sounds a bit "immature".
My I suggest more of a business kind of language here?

```diff
- – it's very important
+ – Adherence to our core values is critical to successfully passing the interview process and the probation period.
```

Reply:
@hesalx well, the text you've suggested won't get anyone's attention. It's good for other purposes, not for the vacancy. You can't catch a tiger on a dead worm.

Here is the fact, the original text got you.
See, we've stuck there discussing it – this is exactly how I want the vacancy text to work.

Do you have any facts that prove that this text is not good for the vacancy except your opinion that it's immature?

Reply:
We are discussing it but I am not a candidate. The original text got me because I myself would be much more cautious about such a vacancy and more likely to pass by even despite the fact that the values themselves are the reason we are having this conversation.
That's all I have to say in the matter.

Reply:
@hesalx I respect your point of view however, I kinda disagree with you. For me, it looks like your proposal is rather turning around the personal taste, not about something objectively good that can be proven.

Sorry if it sounds assertive or anything, don't get me wrong, it's actually great that either of us can have our personal option. In this very situation, I'd prefer to continue going with mine because I understand it better. I'll totally be happy to change closer to your suggestion if my original one doesn't work.